src/app/verification/page.tsx

```diff
@@ -0,0 +1,26 @@
+import ControlOtp from '@/components/pages/verification/control-otp'
+import Content from '@/components/ui/content'
+import { Icons } from '@/components/ui/icons'
+import ImageCard from '@/components/ui/image-card'
+import { ThemeToggler } from '@/components/ui/theme-toggler'
+
+export default function Verification() {
+  return (
+    <section className="h-screen w-screen bg-gradient-blue-white">
```

Comment:
For each existing page we just keep copying <section><Content></..></..> with same classes. I think it'll be better if we:

1.  replace section tag 'cause it is [semantically wrong](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#:~:text=Sections%20should%20always%20have%20a%20heading)
2. create sofisticated component a.k.a AuthContainer(as example)

Reply:
`main` or `div`

Reply:
We already have `main` in the `layout.tsx`. Also, we have an `<h1>` tag so I don't see any problems using section

Reply:
Oh, I thought that we have `main` in the `layout.tsx`. However, in case we'll have it there, there won't be any serious problems, cause we have headings on each auth page (so `section` is suitable here)

Reply:
So let's add `main` to the `layout.tsx`. Oh, as far I know, `section` should have a heading as a child of the `section` element, so it's better to avoid `section` tag and use default div(read the [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section))

Reply:
Semantics is always a pain in the *** 😄 I'll change some tags than

Reply:
Yes, semantic is quite _holivarna_ topic 🍔